test/uk/gov/hmrc/tai/calculators/EstimatedPayCalculatorSpec.scala

```diff
@@ -95,7 +96,12 @@ class EstimatedPayCalculatorSpec extends PlaySpec {
     "calculate net and gross pay amounts which are apportioned for the current year" when {
 
       "an employment start date is supplied which falls in the current tax year" in {
-        val startDateHalfThroughYear = TaxYear().end.minusDays(182)
+
+        val startDateHalfThroughYear = if (new LocalDate(TaxYear().year + 1,1,1).year().isLeap) {
+          TaxYear().end.minusDays(183)
+        } else {
+          TaxYear().end.minusDays(182)
+        }
         val payDetails = PayDetails(paymentFrequency = PayFreq.weekly, pay = Some(100),
```

Comment:
Is this better as something like: Days.daysBetween(startDate, endDate).getDays()